### DISCUSSION TITLE
IfcSweptDiskSolid not in AbRV

### DISCUSSION THREAD

**Discussion by RickBrice** [jwouellette/TPF-5_372-Unit_Test_Suite#3] [General]
Jeff - Something to bring up to the IFC specification team.

IfcSweptDiskSolid is not part of the AbRV
![image](https://github.com/jwouellette/TPF-5_372-Unit_Test_Suite/assets/37087370/47aa5047-e8c5-4462-8cf3-e8b9ad6ca74b)

However, the examples in 7.11.3.10.7 for reinforcing stirrup and reinforcing assembly both use IfcSweptDiskSolid for the representation of reinforcing bars. 

Should IfcSweptDiskSolid be part of the AbRV?

**Comment by JOuellette-Autodesk** [comment]
I think it should be and we should insist it is included via the bSI IFC4.x IF. Submit an issue and I'll chime in.

**Comment by JOuellette-Autodesk** [comment]
Also, Evandro Alfieri of bSI indicated that the AbRV is NOT yet "final" and that would be the job of the IFC4.x IF to determine.
